src/chapters/configuring.xml

```diff
@@ -2772,6 +2794,24 @@ ed272546-87bd-4db9-acba-e36e1a9ca20a
                                         session will use the user's default shell.</para>
                                 </entry>
                             </row>
+                            <row>
+                                <entry><parameter>server-alive-interval</parameter></entry>
+                                <entry>
+                                    <para>
+                                        <indexterm>
+                                            <primary>SSH</primary>
+                                            <secondary>server-alive-interval</secondary>
+                                        </indexterm>
+                                        By default the SSH client does not send keepalive requests
+                                        to the server.  This parameter allows you to configure the
+                                        the interval (in seconds) at which the client connection
+                                        sends keepalive packets to the server.  The default is 0,
+                                        which disables sending the packets.  A value of 1 is
```

Comment:
> ... A value of 1 is automatically increased to 2.  Negative values are set to 0, disabling sending keepalives.

The "default is zero, one becomes two, negative becomes zero" description seems convoluted/confusing. It makes sense in the code, but this description borders on being pseudocode.

Wouldn't it be easier to understand as simply being disabled by default, with the minimum legal value being 2?